lib/MentionsForm/MentionsFormActions.js

```diff
@@ -17,6 +17,14 @@ const MentionsFormActions = props => {
     ? ['collapse', 'link-default']
     : ['primary'];
 
+  const peopleText =
+    props.mentionCount === 1
```

Comment:
We could have a helper function that either gives us back `person/people` based on a number input

Reply:
Yeah this is a tricky one, the helper exists in app, but i wasn't sure wether to add it here too. I think it makes sense to make a helpers file though